docs/site/src/components/ApiMenu.js

```diff
@@ -0,0 +1,68 @@
+// @flow weak
+
+import React, { PureComponent, PropTypes } from 'react';
+import { camelCase } from 'docs/site/src/utils/helpers';
```

Comment:
Add that the the MUI code style guide. 😄 

I did wonder what the order should be, and there was method to my madness, but yours is better.